crates/viewer/re_renderer/src/video/chunk_decoder.rs

```diff
@@ -53,70 +51,86 @@ impl re_byte_size::SizeBytes for DecoderOutput {
 pub struct VideoSampleDecoder {
     debug_name: String,
     decoder: Box<dyn re_video::AsyncDecoder>,
-    decoder_output: Arc<Mutex<DecoderOutput>>,
+
+    decoder_output_receiver: crossbeam::channel::Receiver<re_video::DecodeResult<Frame>>,
+    decoder_output: DecoderOutput,
```

Comment:
This is confusing. `decoder_output_receiver` is receiving output (confusing in itself), and then storing it in `decoder_output`?